paddle/fluid/operators/compat/fused_conv2d.pbtxt

```diff
@@ -39,14 +39,40 @@ def {
     name: "data_format"
     type: STRING
   }
+}
+extra {
   attrs {
     name: "fuse_activation"
     type: STRING
   }
+  attrs {
+    name: "fuse_alpha"
+    type: FLOAT
+  }
+  attrs {
+    name: "fuse_beta"
+    type: FLOAT
+  }
   attrs {
     name: "fuse_residual_connection"
     type: BOOLEAN
   }
+  attrs {
+    name: "Scale_in"
+    type: FLOAT
+  }
+  attrs {
+    name: "Scale_out"
+    type: FLOAT
+  }
+  attrs {
+    name: "Scale_in_eltwise"
+    type: FLOAT
+  }
+  attrs {
+    name: "Scale_weights"
+    type: FLOATS
+  }
```

Comment:
There can also be "Bias_scales" optionally.

Reply:
Done. Thanks~